using-mobile-extensions/mobile-core/configuration/README.md

```diff
@@ -0,0 +1,65 @@
+# Configuration
+
+The Configuration extension is built into the Mobile Core.  With its APIs you will generate the necessary events in the Event Hub that will provide configuration options for all installed extensions.
```

Comment:
In the second sentence, please change "you will generate" to "you can generate",  change ""that will provide" to "that provide", and add "the" between "provide" and "configuration". Please also check the spacing after the period.

The sentence should read as  follows:

"With its APIs, you generate the necessary events in the Event Hub that provide the configuration options for all installed extensions."